lib/core/lib/core/models/opportunity/base.tsp

```diff
@@ -40,26 +39,28 @@ using CommonGrants.Types;
  * }
  * ```
  */
-@doc("A funding opportunity") // Overrides internal docstrings when emitting OpenAPI
+@doc("A funding opportunity")
 model OpportunityBase {
   /** Globally unique id for the opportunity */
   @visibility(Lifecycle.Read)
   id: uuid;
 
   /** Title or name of the funding opportunity */
+  @example("Small business grant program")
```

Comment:
Without this the OpenAPI spec just displays "string" 

The reason we don't have an example at the model level is that it causes a compilation error if people extend the `OpportunityBase` model and add their own custom fields.